apps/condo/domains/acquiring/access/Payment.js

```diff
@@ -7,7 +7,8 @@ const { checkOrganizationPermission } = require('@condo/domains/organization/uti
 const { checkAcquiringIntegrationAccessRight } = require('../utils/accessSchema')
 
 const { throwAuthenticationError } = require('@condo/domains/common/utils/apolloErrorFormatter')
-const get = require('lodash/get')
+const { get } = require('lodash')
```

Comment:
no need to import whole lodash when you need only one method. :)

Reply:
Hmm, yes. Fixed.